lib/tenstorrent/bh_arc/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ zephyr_library_sources(
   post_code.c
   read_only_table.c
   smbus_target.c
-  spi_controller.c
+  #spi_controller.c
```

Comment:
This should probably be removed rather than commented.

Reply:
Fixed